src/azul/service/manifest_service.py

```diff
@@ -613,9 +613,23 @@ def __init_subclass__(cls) -> None:
             assert format not in cls.cls_for_format
             cls.cls_for_format[format] = cls
 
+    @classmethod
+    def _cmd_exe_quote(cls, s: str) -> str:
+        """
+        Escape a string for insertion into a `cmd.exe` command line
+        """
+        assert '"' not in s, s
+        assert '\\' not in s, s
+        return f'"{s}"'
+
     @classmethod
     def command_lines(cls, url: str) -> Optional[JSON]:
-        return None
+        return {
+            # -L flag follows redirects; -O to write output to a local file & -J
+            # tells -O to use the server-specified Content-Disposition filename
+            'cmd.exe': f'curl.exe -L {cls._cmd_exe_quote(url)}',
```

Comment:
No `-O` or `-J` for Windows command-line? If so I think this deserves a comment explaining why they aren't needed for Windows but are for Bash.

Reply:
I tried to be direct with the comment, to stay within the …
>[ explaining briefly (!) ](https://github.com/DataBiosphere/azul/pull/3183#discussion_r658406717)

… indication, and also not too thorough since it might have been foreseen …
> [and test whether that works](https://github.com/DataBiosphere/azul/issues/3142#issuecomment-861745753)

… that it might not work. LMKWYT

Reply:
Looks good.